code/modules/modular_computers/file_system/programs/robocontrol.dm

```diff
@@ -3,7 +3,7 @@
 	filename = "robocontrol"
 	filedesc = "Bot Remote Controller"
 	category = PROGRAM_CATEGORY_ROBO
-	program_icon_state = "robot"
```

Comment:
does robot not exist?

Reply:
No. There were a lot that didn't exist previously like the one for AI, the one for Arcade, and some others I cant remember.

For those I created icons.
But I didn't want to do it for this one right now as Software will be the theme of a future Phase.
Also I had on idea better than making it very similar to generic.